### PR TITLE
Feature/view pager UI

### PR DESCRIPTION
**Pull request by gracenudze**

### CODE REVIEW

File: app/src/main/java/com/example/firstapplication/Activity/CryptoHomepageActivity.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            @Override
+            public void onPageScrolled(int position, float positionOffset, int positionOffsetPixels) {
+
+            }
+
+            @Override
+            public void onPageSelected(int position) {
+                createDots(position);
+                if (position == layouts.length -1 ) {
+                    buttonNext.setText("Start");
```
Comment (Knexis) [inline app/src/main/java/com/example/firstapplication/Activity/CryptoHomepageActivity.java:60]: Avoid hard coded "Strings" move 'Start' and 'Next' strings into the strings.xml

File: app/src/main/java/com/example/firstapplication/Activity/CryptoHomepageActivity.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+            R.layout.crpyto_firstslide,R.layout.crypto_secondslide,
+            R.layout.crypto_thirdslide};
+    private MpagerAdapter mpagerAdapter;
+
+    //sliding dots
+    private LinearLayout dots_layout;
+    private ImageView dots[];
+
+    //buttons
+    private Button buttonNext, buttonSkip;
```
Comment (Knexis) [inline app/src/main/java/com/example/firstapplication/Activity/CryptoHomepageActivity.java:30]: nit: rename buttonNext and buttonSkip to btnNext and btnSkip

File: app/src/main/java/com/example/firstapplication/Activity/CryptoHomepageActivity.java [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+public class CryptoHomepageActivity extends AppCompatActivity implements View.OnClickListener {
+    private ViewPager mPager;
+    private int layouts[] = {
+            R.layout.crpyto_firstslide,R.layout.crypto_secondslide,
+            R.layout.crypto_thirdslide};
+    private MpagerAdapter mpagerAdapter;
+
+    //sliding dots
+    private LinearLayout dots_layout;
```
Comment (Knexis) [inline app/src/main/java/com/example/firstapplication/Activity/CryptoHomepageActivity.java:26]: rename to layoutDots

Comment (gracenudze) [reply app/src/main/java/com/example/firstapplication/Activity/CryptoHomepageActivity.java:26]: Yes please

File: app/src/main/java/com/example/firstapplication/Activity/CryptoHomepageActivity.java (hunk 2 above)
Comment (gracenudze) [reply app/src/main/java/com/example/firstapplication/Activity/CryptoHomepageActivity.java:30]: okay

File: app/src/main/java/com/example/firstapplication/Activity/CryptoHomepageActivity.java (hunk 1 above)
Comment (gracenudze) [reply app/src/main/java/com/example/firstapplication/Activity/CryptoHomepageActivity.java:60]: alright

**Comment by Knexis** [comment]
From the impressions this needs to be a full screen page.

image

**Comment by Knexis** [comment]
The title and description texts need to be aligned on the left and right.

image

**Comment by Knexis** [comment]
Increase the margin between the description and the bottom action buttons (Skip and Next)

image

image

**Comment by Knexis** [comment]
Decrease the description text size 
image

**Comment by gracenudze** [comment]
> The title and description texts need to be aligned on the left and right.
> 
> image

it is already aligned on the left and right

**Pull request merged**